_posts/2018-07-17-its-a-lot-more-snappy.md

```diff
@@ -131,13 +127,13 @@ Now that we had implemented this pattern, we noticed that the initial download o
 
 It was at this point our fearless tech lead, William Sia(TM) proposed using a web worker for that exact purpose. This was new and very exciting. Of course we wanted to use a web worker!
```

Comment:
Only concern here is the message it sends out - we chose it for the tech, not for the problem solved.

Partially a question of style, but consider the audience and the impression it leaves.